nixos/modules/services/web-apps/akkoma.md

```diff
@@ -0,0 +1,136 @@
+# Akkoma {#module-services-akkoma}
+
+[Akkoma](https://akkoma.dev/) is a lightweight ActivityPub microblogging server forked from Pleroma.
+
+## Service configuration {#modules-services-akkoma-service-configuration}
+
+The Elixir configuration file required by Akkoma is generated automatically from
+{option}`services.akkoma.config`. Secrets must be included from external files outside of the Nix
+store by setting the configuration option to an attribute set containing the attribute
+{option}`_secret` – a string pointing to the file containing the actual value of the option.
+
+For the mandatory configuration settings these secrets will be generated automatically if the
+referenced file does not exist during startup, unless disabled through
+{option}`services.akkoma.initSecrets`.
+
+The following configuration binds Akkoma to the local IPv6 address `::1` on port 4000, expecting
+to be run behind a HTTP proxy on `fediverse.example.com`.
+
+
+```nix
+services.akkoma.enable = true;
+services.akkoma.config = {
+  ":pleroma" = {
+    ":instance" = {
+      name = "My Akkoma instance";
+      description = "More detailed description";
+      email = "admin@example.com";
+      registration_open = false;
+    };
+
+    "Pleroma.Web.Endpoint" = {
+      url.host = "fediverse.example.com";
+      http = {
+        ip = "::1";
+        port = 4000;
+      };
+    };
+  };
+};
+```
+
+Please refer to the [configuration cheat sheet](https://docs.akkoma.dev/stable/configuration/cheatsheet/)
+for additional configuration options.
+
+The service can be enabled with the usual
+
+```ShellSession
+$ nixos-rebuild switch
+```
```

Comment:
```suggestion
```

The service is enabled by ``services.akkoma.enable = true;`` and we do not need to explain and assume how the nixos system is build here.